libs/components/theme/src/lib/styles/_public-api/_compat/_mixins.scss

```diff
@@ -1,8 +1,18 @@
 // Export variables for backward compatibility.
 // (Some stylesheets only import the mixins.scss file and expected the variables to be available.)
+// TODO: Remove this forward in a breaking change.
 @forward 'variables';
 
 @use '../variables' as *;
+@use 'variables' as *;
+
+@mixin sky-deemphasized {
```

Comment:
was this intentionally added or left in from testing or something?

Just is different from all the other changes so wanted to double check

Reply:
Yes, good question. The modern theme "compat" files need this mixin, but it shouldn't be exported to the "normal" public API, so I moved it to the compat file. Just a consequence of SASS old `@import` structure: everything gets pulled in when it shouldn't; the `sky-deemphasized` mixin "leaked into" the public API indirectly.